### PR TITLE
Fix 3773 - Issues with sending email in 7.9.1

### PR DESCRIPTION
**Pull request by daniel-samson**
issue: #3773 
tasks: scrm-628

## Description

To field is not populated when relating a account/contact in the email compose view.

## Motivation and Context

* Adds missing or desired feature

## How To Test This

* Repair and rebuild
* repair minified js
* repair js groupings
* view emails module
* compose email
* relate email to an account / contact

The to field should be populated with the email1 field

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: config_override.php [hunk 1]
```diff
@@ -1,3 +1,7 @@
 <?php
 /***CONFIGURATOR***/
+$sugar_config['developerMode'] = true;
```
Comment (gymad) [inline config_override.php]: Do we need any config override option in the repo?

File: config_override.php [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+$sugar_config['default_module_favicon'] = false;
+$sugar_config['dashlet_auto_refresh_min'] = '30';
+$sugar_config['stack_trace_errors'] = false;
```
Comment (gymad) [inline config_override.php]: Do we need any config override option in the repo?

File: jssource/src_files/include/javascript/popup_parent_helper.js [hunk 3]
```diff
@@ -1,9 +1,10 @@
-/*********************************************************************************
+/**
```
Comment (gymad) [inline jssource/src_files/include/javascript/popup_parent_helper.js]: js source file modified but not rebuilded and commited into git repo

Comment (daniel-samson) [inline jssource/src_files/include/javascript/popup_parent_helper.js]: Committed minified versions

File: include/SugarFields/Fields/Parent/SugarFieldParent.php [hunk 4]
```diff
@@ -53,22 +59,34 @@ function getDetailViewSmarty($parentFieldArray, $vardef, $displayParams, $tabind
...
+            'id' => $vardef['id_name'],
+            'name' => $vardef['name'],
+        );
+
+        if(isset($displayParams['field_to_name_array']) && !empty($displayParams['field_to_name_array'])) {
+            $popup_request_data['field_to_name_array'] = array_merge(
+            	$popup_request_data['field_to_name_array'],
+                $displayParams['field_to_name_array']
+			);
+		}
```
Comment (daniel-samson) [inline include/SugarFields/Fields/Parent/SugarFieldParent.php]: Reformated code

**Pull request merged**